src/DynamoCore/Models/WorkspaceModel.cs

```diff
@@ -233,7 +233,7 @@ protected virtual void OnConnectorDeleted(ConnectorModel obj)
         /// <summary>
         ///     A set of input parameter states, this can be used to set the graph to a serialized state.
         /// </summary>
-        public IEnumerable<PresetModel> Presets { get { return presets;} }
+        public List<PresetModel> Presets { get { return presets;} }
```

Comment:
This change doesn't work.  :( A  List is a mutable type (i.e. you can call Add on it).  IEnumerable is not.  This type was chosen specifically for this reason.